war/src/main/resources/templates/scores/details.html

```diff
@@ -1,124 +1,134 @@
-                </td>
-                <td><code class="label" data-th-text="${scoreDetail.key}"></code></td>
-                <td align="center" data-th-text="${scoreDetail.value}"></td>
-                <td align="center" data-th-text="${scoreDetail.weight}"></td>
+              <td>
+                <ion-icon data-collapse name="chevron-expand"></ion-icon>
+              </td>
+              <td>
+                <ion-icon data-th-if="${scoreDetail.value} == 100" class="success" name="success"></ion-icon>
+                <ion-icon data-th-unless="${scoreDetail.value} == 100" class="failure" name="failure"></ion-icon>
+              </td>
+              <td><code class="label" data-th-text="${scoreDetail.key}"></code></td>
+              <td align="center" data-th-text="${scoreDetail.value}"></td>
+              <td align="center" data-th-text="${scoreDetail.weight}"></td>
             </tr>
-            <tr class="collapse-content" data-th-classappend="${scoreDetail.value} != 100 ? 'show':_" data-th-id="'collapse-' + ${scoreDetail.key}">
-                <td colspan="5">
-                    <h3>Details:</h3>
-                    <table class="table">
-                        <thead>
-                        <tr>
-                            <th></th>
-                            <th>Value</th>
-                            <th>Weight</th>
-                            <th>Message</th>
-                            <th>Help</th>
-                        </tr>
-                        </thead>
-                        <tbody>
-                        <tr data-th-each="component : ${scoreDetail.componentsResults}">
-                            <td>
-                                <ion-icon data-th-if="${component.score} == 100" class="success" name="success"></ion-icon>
-                                <ion-icon data-th-unless="${component.score} == 100" class="failure" name="failure"></ion-icon>
-                            </td>
-                            <td data-th-text="${component.score}"></td>
-                            <td data-th-text="${component.weight}"></td>
-                            <td>
-                                <ul>
-                                    <li data-th-each="reason: ${component.reasons}" data-th-text="${reason}"/>
-                                </ul>
-                            </td>
-                            <td>
-                                <ul>
-                                    <li data-th-each="resolution: ${component.resolutions}">
-                                        <a data-th-href="${resolution.link}" data-th-text="${resolution.text}"></a>
-                                    </li>
-                                </ul>
-                            </td>
-                        </tr>
-                        </tbody>
-                    </table>
-                </td>
+            <tr class="collapse-content" data-th-classappend="${scoreDetail.value} != 100 ? 'show':_"
```

Comment:
I don't understand why you had to change the indentation of this.

Reply:
I formatted the content via an online editor that split the long length line into smaller lines.  Do i need to revert the changes ?